.github/workflows/publish-docker-latest.yml

```diff
@@ -2,21 +2,30 @@
 on:
   release:
     types: [released]
+  workflow_dispatch:
```

Comment:
```suggestion
```

Are we sure we want to keep this in production?
A new docker image should only be made when a release is made I guess 🤔

Reply:
I added in the description of the PR

> Add `workflow_dispatch` event in case we need to re-trigger it after a failure without creating a new release

In the past, sometimes, we had issues with the docker CI that failed, not related to MeiliSearch, but to the Dockerfile or the CI itself. Because we cannot delete the MeiliSearch release and re-create the same one, I built the docker image on my machine and pushed it to DockerHub manually.
Since this new build of Docker image, I'm not sure I'm able to reproduce it locally since I don't have all the required docker tools on my machine. So I need a way to re-trigger the CI without recreating a release

Why can't I delete and re-create a release for MeiliSearch?
Once a release (not for a RC) is created, it triggers a lot of CIs, and among them, the CI to update Brew and APT package. Once these CIs are triggered, it's not really possible to roll back.

Reply:
But I realized, in any case, I'm stuck... 
Let's say the CI to push v0.24.0 docker image fails. If I want to re-trigger the same CI, I need to provide a `tag`, and the v0.24.0 is already taken. So, either I create a new release, or I find a way to build it manually on my machine...
So you're right, there is no point to let the `workflow_dispatch`